python/ray/autoscaler/updater.py

```diff
@@ -269,9 +270,11 @@ def run(self,
             if exit_on_fail:
                 quoted_cmd = " ".join(final_cmd[:-1] + [quote(final_cmd[-1])])
                 raise click.ClickException(
-                    "Command failed: \n\n  {}\n".format(quoted_cmd))
+                    "Command failed: \n\n  {}\n".format(quoted_cmd)) from None
             else:
-                raise
+                raise Exception(
+                    "SSH command Failed. Look above to see the output from the"
+                    " failure.") from None
```

Comment:
why not just raise the same above clickexception?

Reply:
ClickException masks the traceback, which generally is not useful.

Reply:
Sure!

Reply:
Wait, the traceback still appears with the ClickException.

Reply:
Preventing the traceback somehow would be great. Otherwise this looks like a nice improvement to me

Reply:
Hidden! I also switched the order so that the last thing printed is the corrrect one

Reply:
@edoakes  & @richardliaw I updated the description with the current view. There is currently *no* Python Exception being printed--just the error message.

Reply:
nice!